src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx

```diff
@@ -58,9 +59,9 @@ For users on the [New Relic One user model](/docs/accounts/original-accounts-bil
 
 ## Overview of user management concepts [#understand-concepts]
 
-To optimally use our user management features, it's important to first understand the concept of the "access grant." An access grant gives a group of users access to a) a role and b) an account. For a New Relic organization that has many accounts, groups typically require more than one access grant because users in a group usually need access to multiple accounts and roles. The diagram below explains the elements that make up an access grant.
+To optimally use our more advanced user management features, it's important to first understand the concept of the "access grant." An access grant gives a group of users access to a) a role and b) an account. For a New Relic organization that has many accounts, groups typically require more than one access grant because users in a group usually need access to multiple accounts and roles. The diagram below explains the elements that make up an access grant.
 
-Note that if you're simply adding a user to a [default group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#groups) (**Admin** or **User**) in the account you're in, you don't need to create an access grant: you would simply [add a user](#add-users) to that group and that group already has an associated access grant. But if you're trying to grant users access to a custom group, a custom role, or to other accounts, you must create an access grant.
+Note that if you're on our Standard tier and want to assign a user to a [default group](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#groups) (**Admin** or **User**), you don't need to create an access grant: you would simply [add a user](#add-users) to that group and you're done. But for Pro and Enterprise tier accounts, if you're trying to grant users access to a custom group, a custom role, or to other accounts, you must create an access grant.
```

Comment:
Seems like you could cut "and you're done"

Reply:
i'll get that later, thanks